brew/Brewfile.symlink

```diff
@@ -0,0 +1,36 @@
+tap "homebrew/bundle"
+tap "homebrew/services"
+brew "bat"
+brew "broot"
+brew "defaultbrowser"
+brew "deno"
+brew "fd"
+brew "figlet"
+brew "fish"
+brew "fzf"
+brew "gh"
+brew "git-machete"
+brew "glow"
+brew "ijq"
+brew "jq"
+brew "make"
+brew "mas"
+brew "mosh"
```

Comment:
Also purely a bonus, unless `ssh` isn't getting it done for you. This shows the most value when working and maintaining sessions on remote on servers.